mm2src/mm2_main/tests/mm2_tests/mm2_tests_inner.rs

```diff
@@ -7927,3 +7950,342 @@ fn test_eth_swap_negotiation_fails_maker_no_fallback() {
     block_on(wait_for_swap_negotiation_failure(&mm_bob, &uuids[0], wait_until));
     block_on(wait_for_swap_negotiation_failure(&mm_alice, &uuids[0], wait_until));
 }
+
+#[derive(Debug, Deserialize)]
+#[serde(deny_unknown_fields, tag = "status", content = "details")]
+pub enum InitTrezorStatus {
+    Ok(InitHwResponse),
+    Error(Json),
+    InProgress(Json),
+    UserActionRequired(Json),
+}
+
+pub fn mm_ctx_with_trezor(conf: Json) -> MmArc {
```

Comment:
For test helper functions, please make them async and use block_on where they are called.

Reply:
done